changelog/changelog2020.md

```diff
@@ -73,7 +73,7 @@ However we'd like to ask you to update your clients as soon as possible, we're n
 
 * Added [Resource](../operations/resources.md#resource) that is replacing the Space. It can have multiple categories through multiple services. Note that Resource does not need to be assigned to any category.
 * Added [Resource category](../operations/resources.md#resource-category) that is replacing the Space category.
-* Added [Resource category image assignment](../operations/resources.md#resource-category-image-assignment) that links an image to a category.
+* Added [Resource category image assignment](../operations/resourcecategories.md#resource-category-image-assignment) that links an image to a category.
 * Added [Resource category assignment](../operations/resources.md#resource-category-assignment) that links a resource to a category. Note that resource can be assigned only to a single category within the same service.
 * Added [Resource feature](../operations/resources.md#resource-feature) that is replacing the Space feature and now is within the service context.
 * Added [Resource feature assignment](../operations/resources.md#resource-feature-assignment) that is replacing the Space feature assignment.
```

Comment:
Some of the references to the moved entities were not updated. There probably are others too, would be nice to check.

Reply:
I don't know whether it is better to update historic references, or not? I can see arguments for and against. But if we do, then yes we should do them all.